### PR TITLE
New amber guides and fix some NPM vulnerabilities

### PR DESCRIPTION
**Pull request by faustinoaq**
Hi @amberframework/contributors 

This PR adds a new link for our new amber guides on the new shiny gitbook ui :sparkles: 

Please see https://amberframework.gitbook.io/amber/

Also I did a quick update to fix NPM packages vulnerabilities

I already working on remove some packages, I will open a new PR for that :wink:

### CODE REVIEW

**Comment by faustinoaq** [comment]
This PR will solve this message:

screenshot_20180514_184047

**Comment by faustinoaq** [comment]
screenshot_20180514_083405

screenshot_20180514_083452

[New guides](https://amberframework.gitbook.io/amber) looks awesome!!!, Thank you @GitbookIO :heart_eyes:

File: .gitignore [hunk 1]
```diff
@@ -5,11 +5,10 @@
 /.vscode/
 /tmp/
 /log/
+/bin/
 .env
 .amber_secret_key
 .encryption_key
 .DS_Store
 node_modules
-amberframework
```
Comment (elorest) [inline .gitignore]: These were added in the past because people had committed these binaries. Hopefully they won't be created again now though.

Comment (faustinoaq) [reply .gitignore]: These binaries are still on this repo history, I think we should remove them using some tool or recreate this repo again after merging these PRs :sweat_smile: 

screenshot_20180514_183914

This took me several minutes to push and pull :sweat_smile:

Comment (faustinoaq) [reply .gitignore]: Well, I gonna try this tool: https://rtyley.github.io/bfg-repo-cleaner/ :sweat_smile:

Comment (faustinoaq) [reply .gitignore]: > The problem is that Github also saves branches after you merged, closed and deleted them. You have no way to modify those branches. 

Oh, seems I can't remove big files in this repo after all :disappointed: 

Ref: https://jorin.me/clean-git-history/

I wanted to remove these files (not required anymore)

screenshot_20180514_222747

**Pull request merged**